docs/UI/Slider/index.md

```diff
@@ -29,6 +29,7 @@ Purpose
 |Name|Type|Mandatory|Additional|
 |:---|:---|:--------|:---------|
 |sliderPosition|Integer|false|minvalue: 1<br>maxvalue: 26|
+|cancelID|Integer|false|can be used later with CancelInteraction RPC to specifically close this slider|
```

Comment:
This table does not have a description section

Reply:
fixed in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/163/commits/1f93da0ff41d60f00c4d1bb3cc93ffeb5bd034e0